idaes/core/util/misc.py

```diff
@@ -132,6 +131,16 @@ def copy_port_values(destination=None, source=None, arc=None,
             direction=direction)
 
 
+def svg_tag(*args, **kwargs):
+    """
+    Moved to idaes.core.util.tags.svg_tag
+    Leaving redirection function here for deprecation warning.
+    """
+    _log.warning("DEPRECATED: idaes.core.util.misc.svg_tag has been deprecated. "
```

Comment:
I would suggest using the Pyomo deprecation warning here: from pyomo.common.deprecation import deprecation_warning

This lets you tag the version that is was deprecated in and the version it will be removed in.

Otherwise, I think you should make a note of when this deprecation was added so we can judge when to finally remove it.